src/parser.cpp

```diff
@@ -376,7 +376,7 @@ void Parser::ParserImpl::loadModel(const ModelPtr &model, const std::string &inp
     }
 
     if (!encapsulationNodes.empty()) {
-        loadEncapsulation(model, encapsulationNodes[0]);
+        loadEncapsulation(model, encapsulationNodes.at(0));
```

Comment:
Why?

Reply:
Not related to this PR, was accidental when searching for `.at`

Reply:
Ok, fair enough, although it didn't harm changing it (since the call to `.at()` doesn't account for a potential exception). Ok, added this to issue #402.

Reply:
I know ... but I want this to go through as fast as poss, so I figured it was not the time to address every little thing I found along the way! Glad we have an issue for general stuff, I'll add some things there too.